plugins/nextcloud-rule-exclusions-before.conf

```diff
@@ -1531,8 +1531,8 @@ SecRule REQUEST_FILENAME "@rx /ocs/v[0-9]+\.php/apps/user_status/api/v[0-9]+/use
 # [ Nextcloud Deck ]
 #
 
-# Moving a card in Deck app
-SecRule REQUEST_FILENAME "@rx /apps/deck/cards/[0-9]+/reorder$" \
+# When updating a card
+SecRule REQUEST_FILENAME "@rx /apps/deck/cards/[0-9]+" \
```

Comment:
You should try to match as exact as you can for the URL path to minimize the chance of a bypass. Based on the URLs provided in your regression test, you should use this regular expression.
```suggestion
# When updating a card
# Moving a card in deck
# Unassigning a user from a card
# Setting a due date on a card
SecRule REQUEST_FILENAME "@rx /apps/deck/cards/[0-9]+(?:/reorder|/unassign)?$" \
```
https://regexr.com/7s4pl